Documentation/SystemModules.md

```diff
@@ -98,7 +101,8 @@ First create a directory called `CJasPer` parallel to `CJPEG` and our example ap
     CJPEG$ cd ..
     $ mkdir CJasPer
     $ cd CJasPer
-    CJasPer$ touch module.modulemap Package.swift
+    CJasPer$ swift package init
```

Comment:
This isn't correct, `swift package init` will add other files which are not appropriate in a system module map package.